examples/time_frequency/plot_time_frequency_erds.py

```diff
@@ -48,6 +48,7 @@
 from mne.datasets import eegbci
 from mne.io import concatenate_raws, read_raw_edf
 from mne.time_frequency import tfr_multitaper
+from mne.stats import permutation_cluster_1samp_test as pctest
```

Comment:
Do we use the same abbrev. elsewhere?

Reply:
I don't think so, but this function name is terrible (30 characters, there should be a PEP8 rule).

Reply:
we do not abbrev in other examples IIRC but maybe `cluster_test` or `permutation_test`?

Reply:
What's wrong with `pctest` then if we don't use abbreviations anywhere else? It's both a cluster and a permutation test :smile:.

Reply:
It's a test how politically correct you are

Reply:
and mac users may be offended by `pctest`